cmd/harbor-scanner-clair/main.go

```diff
@@ -35,7 +35,10 @@ func main() {
 		log.Fatalf("Error: %v", err)
 	}
 
-	store := redis.NewStore(config.Store)
+	store, err := redis.NewStore(config.Store)
+	if err != nil {
+		panic(err)
```

Comment:
```suggestion
		log.Fatalf("Error: %v", err)
```

Reply:
this error is just for url mistaken, if we do not panic here, the service will startup, and may function error in future.
So I suggest we should fail earlier and explicitly for such config error.

Reply:
`log.Fatalf` logs an error and exits with status code 1. So there's no way to start the service in case of errors